models/mlp_node_feature.py

```diff
@@ -0,0 +1,50 @@
+import torch
+from torch import Tensor
+from torch.nn import ModuleList, Sequential, ReLU, Linear, Module
+
+
+class mlp_node_feature(Module):
```

Comment:
```suggestion
class MLPNodeFeature(Module):
```

Reply:
I would put this in `Base.py`

Reply:
I moved it out from Base.py to make the structure simple.

Reply:
Sorry, I mean in the file, not in the class